commands/DeploymentPlanCommand.js

```diff
@@ -0,0 +1,24 @@
+const BarracksCommand = require('./BarracksCommand');
+
+class DeploymentPlanCommand extends BarracksCommand {
+  validateCommand(program) {
+    return !!(
+        program.packageReference &&
+        program.packageReference !== true &&
+        typeof program.packageReference !== 'function'
```

Comment:
That check is no longer needed (cleanupProgramOptions function does that before calling execute)